src/gui/qjacktrip.cpp

```diff
@@ -856,6 +891,11 @@ void QJackTrip::saveSettings()
     settings.setValue("LastAddress", m_ui->addressComboBox->currentText());
     settings.setValue("ChannelsSend", m_ui->channelSendSpinBox->value());
     settings.setValue("ChannelsRecv", m_ui->channelRecvSpinBox->value());
+#ifdef __RT_AUDIO__
+    settings.setValue("Backend", m_ui->backendBox->currentIndex());
+    settings.setValue("SampleRate", m_ui->sampleRateBox->value());
+    // TODO: save device
+#endif
```

Comment:
It would be good to place this in a separate group.

```
settings.beginGroup("Audio");
settings.setValue("Backend", m_ui->backendBox->currentIndex());
settings.setValue("SampleRate", m_ui->sampleRateBox->value());
settings.endGroup();
```

Maybe have it just above or below the RecentServers group.

Reply:
(And a corresponding change will need to be made in load settings)

Reply:
Done, is that good?